### PR TITLE
docs: vtex integration

### PR DESCRIPTION
**Pull request by icazevedo**
## What's the purpose of this pull request?
This requests add docs on how to integrate with VTEX checkout and introduces the topic for other integrations, such as login, orderPlaced and my account.

### CODE REVIEW

File: docs/src/theme/DocCategoryGeneratedIndexPage/index.js [hunk 1]
```diff
@@ -1,49 +0,0 @@
-/**
```
Comment (lariciamota) [inline docs/src/theme/DocCategoryGeneratedIndexPage/index.js:1]: Did you delete it on purpose?

File: docs/src/theme/DocCategoryGeneratedIndexPage/styles.module.css [hunk 2]
```diff
@@ -1,21 +0,0 @@
-/**
```
Comment (lariciamota) [inline docs/src/theme/DocCategoryGeneratedIndexPage/styles.module.css:1]: Same question about this file

**Review by lariciamota — APPROVED** [review]
Great!!

File: docs/docs/how-to-guides/vtex-integration/introduction.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+
+## How to integrate FastStore with VTEX
+
+FastStore can be integrated with VTEX to provide a great checkout and post-purchase experience. Follow the steps below to know how.
```
Comment (emersonlaurentino) [inline docs/docs/how-to-guides/vtex-integration/introduction.md:4]:
```suggestion
FastStore can be integrated with VTEX to provide a great checkout and post-purchase experience. Follow the steps below to know-how.
```

File: docs/docs/how-to-guides/vtex-integration/introduction.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+:::info
+Your store has to be active and in production for your VTEX integration to work.
+:::
+
+## Step 4 - Update your `store.config.js` file
+
+Configure your store so that it points to the right addresses. In the `store.config.js` file, change the `storeUrl` property so that it points to your main domain and change the `secureSubdomain`, `checkoutUrl`, `loginUrl`, and `accountUrl` property so that they match `https://{subdomain}.{maindomain}`, `https://{subdomain}.{maindomain}/checkout`, `https://{subdomain}.{maindomain}/api/io/login`, and `https://{subdomain}.{maindomain}/api/io/account` respectively.
+
+Here's an example of how this code block would look after configuring a store with main domain as `vtexfaststore.com` and subdomain `secure`:
```
Comment (emersonlaurentino) [inline docs/docs/how-to-guides/vtex-integration/introduction.md:30]:
```suggestion
Here's an example of how this code block would look after configuring a store with the main domain as `vtexfaststore.com` and subdomain `secure`:
```

**Review by emersonlaurentino — APPROVED** [review]

File: docs/docs/how-to-guides/vtex-integration/introduction.md (hunk 3 above)
Comment (lariciamota) [reply docs/docs/how-to-guides/vtex-integration/introduction.md:4]: know-how is the noun, I believe he wanted the verb+adverb

File: docs/src/theme/DocCategoryGeneratedIndexPage/index.js (hunk 1 above)
Comment (icazevedo) [reply docs/src/theme/DocCategoryGeneratedIndexPage/index.js:1]: Yes! It was @carolinamenezes who deleted it!

**Review by victorhmp — APPROVED** [review]

File: docs/docs/how-to-guides/platform-integration/vtex/hosting-a-faststore-vtex-website.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+## Before you start
+
+Before proceeding any further, make sure you have access to a **VTEX** account and have already [developed your storefront project with **FastStore**.](/tutorials/gatsby-overview)
+
+---
+
+## Step by step
+
+By the end of these steps, you'll host your FastStore website on the internet. Hence, if you'll configure other VTEX solutions (**Checkout**, **Order Placed**, **Login**, and **My Account**) afterward, make sure to use a fictitious domain in the following steps. Then, once you set up all the desired integrations, repeat this guide procedure using your final domain to make your website publicly available to end-users.
```
Comment (carolinamenezes) [inline docs/docs/how-to-guides/platform-integration/vtex/hosting-a-faststore-vtex-website.md:20]: @icazevedo 👍🏼?

**Pull request merged**